.circleci/config.yml

```diff
@@ -699,6 +702,43 @@ jobs:
           path: test-artifacts
           destination: test-artifacts
 
+  test-e2e-chrome-rpc:
```

Comment:
A separate CI job for the JSON-RPC tests. I chose Chrome as it's a faster browser to launch and run the tests than Firefox. I don't see any reason why we would need to run these tests cross browsers